app/[network]/tokens/layout.tsx

```diff
@@ -0,0 +1,11 @@
+import { Network } from "@/components/button/types";
+import App from "./app";
+
+interface RootProps {
+  params: { network: Network };
+  children: React.ReactNode;
+}
```

Comment:
export this in a seperate file so you can use in other files too. make `children` optional